core/trino-main/src/test/java/io/trino/server/remotetask/TestHttpRemoteTask.java

```diff
@@ -394,8 +394,8 @@ public void testAdaptiveRemoteTaskRequestSize()
                 .setCatalog("tpch")
                 .setSchema(TINY_SCHEMA_NAME)
                 .setSystemProperty(REMOTE_TASK_ADAPTIVE_UPDATE_REQUEST_SIZE_ENABLED, "true")
-                .setSystemProperty(REMOTE_TASK_MAX_REQUEST_SIZE, "100kB")
-                .setSystemProperty(REMOTE_TASK_REQUEST_SIZE_HEADROOM, "10kB")
+                .setSystemProperty(REMOTE_TASK_MAX_REQUEST_SIZE, "10kB")
```

Comment:
With the new approach of just asserting the number of update requests is ` > 1`, the old values should be fine here. It's possible that the new values are so small that it caused the current test run to fail due to putting too much load on the test container.

Reply:
I reduced the number of spilts at line 409 since the test is no more flaky.